datarobot_provider/hooks/datarobot.py

```diff
@@ -75,7 +78,15 @@ def get_conn(self) -> dr.Client:
         api_key = conn.extra_dejson.get('extra__http__api_key', '')
         if not api_key:
             raise AirflowException("API key is not defined")
-        return Client(api_key, endpoint)
+
+        # Creating version-specific user agent suffix to collect usage statistics and troubleshooting proposes:
```

Comment:
`... for collecting usage statistics and troubleshoot purposes`

Reply:
fixed, thanks